src/components/NewDraftWizard/WizardForms/WizardJSONSchemaParser.js

```diff
@@ -394,18 +401,19 @@ const FormSelectField = ({ name, label, required, options, nestedField }: FormSe
   </ConnectForm>
 )
 
-/*
+/* !!! Analysis Sequence Variantion imputation checkbox does not have description
```

Comment:
yes that is true, if they ever add on in the ENA schemas we can add one as well, but please meanwhile these kind of comments either we create issues in backend or we keep them somewhere else, not in the code.